internal/services/sentinel/sentinel_alert_rule_scheduled_resource.go

```diff
@@ -301,7 +301,7 @@ func resourceSentinelAlertRuleScheduled() *pluginsdk.Resource {
 			"entity_mapping": {
 				Type:     pluginsdk.TypeList,
 				Optional: true,
-				MaxItems: 5,
+				MaxItems: 10,
```

Comment:
Although the error message provided by the API is descriptive and it seems like it makes sense to hand off the validation to the API, I think we should actually keep the `MaxItems` validation in the schema here because there's value in informing the user earlier of any config issues at plan time, instead of at apply time. WDYT?

Reply:
agree!